src/js/plugin/PluginLoader.js

```diff
@@ -24,10 +24,6 @@ const PluginLoader = {
       pluginHost.setAttribute("frameborder","0");
       pluginHost.setAttribute("style",
         "position:absolute; top:-1px; left:-1px;");
-      pluginHost.setAttribute("sandbox","allow-same-origin allow-scripts");
-      pluginHost.setAttribute("src", URL.createObjectURL(new Blob([
-        `<html><head></head><body>${pluginId}</body></html>`
-      ], {type: "text/html"})));
```

Comment:
I guess we don't lose much by removing `allow-same-origin allow-scripts`. In fact, according to [MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element/iframe)

> When the embedded document has the same origin as the main page, it is strongly discouraged to use both `allow-scripts` and `allow-same-origin` at the same time, as that allows the embedded document to *programmatically remove the sandbox attribute*. Although it is accepted, this case is no more secure than not using the sandbox attribute.

and

> Sandboxing in general is only of minimal help if the attacker can arrange for the potentially hostile content to be displayed in the user's browser outside a sandboxed iframe. It is recommended that such content should be served from a separate dedicated domain, to limit the potential damage.

I'm tempted to say we don't need any of those, so the PR lgtm.